pontoon/base/utils.py

```diff
@@ -89,9 +95,11 @@ class SpacesPlaceable(base.Ph):
         parse = classmethod(general.regex_parse)
 
     PARSERS = [
+        NewlineEscapePlaceable.parse,
         TabEscapePlaceable.parse,
         EscapePlaceable.parse,
-        general.NewlinePlaceable.parse,
+        # DISABLED: turns newline character into \n, see Firefox: netError.dtd
```

Comment:
If it's disabled we might as well just remove it rather than commenting it out. It's not like we'll add it back in later and _not_ notice that it just doesn't work right.